src/Services/Export/WB.Services.Export/Services/Processing/DataExportProcessView.cs

```diff
@@ -11,6 +11,7 @@ public class DataExportProcessView
         public DateTime BeginDate { get; set; }
         public DateTime? EndDate { get; set; }
         public DateTime LastUpdateDate { get; set; }
+        public DateTime CreatedDate { get; set; }

```

Comment:
meed backward compatibility for existed records

Reply:
This field existed in DB